package.json

```diff
@@ -13,7 +13,7 @@
     "@emotion/styled": "^11.3.0",
     "@mui/material": "^5.0.0",
     "firebase": "^9.0.2",
-    "next": "11.1.2",
+    "next": "^11.1.2",
```

Comment:
Did you try installing more packages? I can also see that a yarn.lock file was generated, which means that you might have tried adding another package via yarn. Moving forward, if you ever want to add new packages, just use npm. Let's also look into this during work session!